front-end/src/components/Apollo.tsx

```diff
@@ -104,12 +104,14 @@ const Apollo = ( { children }:Props ) => {
 				const name = tokenPayload.name
 				const username =  tokenPayload.username
 				const email = tokenPayload.email
+				const email_verified = tokenPayload.email_verified
```

Comment:
@niklabh  that addition (and the one under) is unrelated but came out when I merged master. I remember mentioning this in a comment. Is that wrong?

Reply:
This is right here. I forget to add this in my recent commit. But i remember fixing this in the morning. Github is acting weird today